atlp-rwanda/hackers-ec-Fe
# #187419004 Users should be able to Login via Google.

#### What does this PR do?

### Users should be able to Login via Google. 

#### How should this be manually tested?

clone into the branch `ft-login-via-google-187419004`
do `npm install`
run `npm run dev` and navigate to home page of our application
click on login to see the form and then click on `continue with google button`

#### What are the relevant pivotal tracker/Trello stories?
187419004
